wai-aria/subtree/basic.tentative.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/wai-aria/scripts/aria-utils.js"></script>
+
+<div id="tablist" role="tablist" aria-label="tablist">
+  <div id="tab1" role="tab" aria-label="tab1"></div>
+  <div id="tab2" role="tab" aria-label="tab2"></div>
+</div>
+
+<script>
+
+promise_test(async t => {
+  const tablist = await test_driver.get_element_accessible_properties(document.getElementById("tablist"));
```

Comment:
When would I write a test this way as opposed to using `VerifyAccessibilitySubtree()`? At first glance, they're testing equivalent things.

Reply:
I suggested the promise tests be abstracted into the utils file methods. From his reply an earlier comment, it sounds like Jamie may be in the process of replacing these.

Reply:
These. 
https://github.com/web-platform-tests/wpt/blob/master/wai-aria/role/basic.html
https://github.com/web-platform-tests/wpt/blob/master/accname/basic.html